apps/glusterfs/operations_block_volume.go

```diff
@@ -486,7 +488,10 @@ func (bve *BlockVolumeExpandOperation) Build() error {
 		}
 
 		requiredFreeSize := bve.newSize - bv.Info.Size
-		if bhv.Info.BlockInfo.FreeSize < requiredFreeSize {
+		if requiredFreeSize == 0 {
+			logger.Info("Re-executing block volume expansion on [%v]: volume not fully expanded on backend", bve.bvolId)
```

Comment:
This log line might give an illusion to the user that there is some kind of retry mechanism. My suggestion would be something like:

"executing block volume expand on [%v] as usable size is not same as the size."

But it is not a hard requirement.

Reply:
I initially had:

```
logger.Info("Attempting to fix failures seen by recent expand operation")
```
The current one is something that @phlogistonjohn suggested.

I personally don't have a preference for this, let me know which one should we go with, from all these 3 type of msgs.

Reply:
@phlogistonjohn which one do you prefer?

Reply:
I would keep "Re-executing" and swap out the stuff after the brackets for something like @raghavendra-talur has suggested. It's the most precise statement: "Re-executing block volume expansion on [%v]: usable size is not same as the size`
`

Reply:
The reason for "Re-executing" is that its a breadcrumb/reminder that Build has basically already done it's thing and nothing new needs to be built but Exec will be redoing what was already attempeted.

Reply:
done :+1: